environment.yml

```
@@ -0,0 +1,55 @@
+name: springtime
+dependencies:
```

Mambaforge has this by default, but for older versions of conda/mamba it might be helpful to add the conda-forge channel here, as most of these packages are not available in defaults.

```suggestion
name: springtime
channels:
  - conda-forge
dependencies:
```